rpm/sailjail.spec

```diff
@@ -75,71 +61,32 @@ Summary: Tests files for %{name}-daemon
 %setup -q -n %{name}-%{version}
 
 %build
-make %{_smp_mflags} \
-  LIBDIR=%{_libdir} \
-  KEEP_SYMBOLS=1 \
-  HAVE_FIREJAIL=%{jailfish} \
-  release pkgconfig
-
-make %{_smp_mflags} \
-  VERSION=%{version}\
-  _LIBDIR=%{_libdir}\
-  _SHAREDSTATEDIR=%{_sharedstatedir}\
-  -C daemon build
-
-make %{_smp_mflags} \
-  VERSION=%{version}\
-  TESTSDIR=%{_testsdir}\
-  -C daemon test-build
+%meson \
+    -Dversion=%{version} \
+    -Duserunitdir=%{_userunitdir} \
+    -Dunitdir=%{_unitdir} \
+    -Ddbuspolicydir=%{_dbuspolicydir} \
+    -Dtestsdir=%{_testsdir} \
+
+%meson_build
 
 %install
 rm -rf %{buildroot}
```

Comment:
I would propose to remove this line as well, as it's useless with modern rpm.

Reply:
Seems so. I removed it now.